src/main/webapp/js/issues-detail.js

```diff
@@ -1,45 +1,39 @@
-/* global jQuery, view */
+/* global jQuery, view, renderZoomableTrendChart */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['jQuery' is defined but never used.](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=4663559)